### PR TITLE
update description of -seqerror

### PR DESCRIPTION
**Pull request by AprilYUZhang**

### CODE REVIEW

**Comment by gregorgorjanc** [comment]
@AprilYUZhang thanks for this 1st PR!

@XingerTang can you please review this PR - it's simple;) I will look when you give me a go ahead;)

**Review by XingerTang — APPROVED** [review]
Looks good to me :)

**Comment by XingerTang** [comment]
> Looks good to me :)

@gregorgorjanc

File: docs/source/usage.rst [hunk 1]
```diff
@@ -123,11 +123,13 @@ Peeling arguments:
 
 For hybrid peeling, where a large amount (millions of segregating sites) of sequence data needs to be imputed, first run the program in multi-locus mode to generate a segregation file, and then run the program in single-locus mode with a known segregation file.
 
-The ``-error``, ``-seqerror`` and ``-length`` arguments control some of the parameters used in the model. |Software| is robust to deviations in genotyping error rate and sequencing error rate so it is not recommended to use these options unless large deviations from the default are known. Changing the ``-length`` argument to match the genetic map length can increase accuracy in some situations.
+The ``-error``, ``-seqerror`` and ``-length`` arguments control some of the parameters used in the model. ``-seqerror`` can not be zero. |Software| is robust to deviations in genotyping error rate and sequencing error rate so it is not recommended to use these options unless large deviations from the default are known. Changing the ``-length`` argument to match the genetic map length can increase accuracy in some situations.
```
Comment (gregorgorjanc) [inline docs/source/usage.rst:126]: @AprilYUZhang I think we best write "must not be zero";)

File: docs/source/usage.rst [hunk 2]
```diff
@@ -123,11 +123,13 @@ Peeling arguments:
...
 
 The ``-esterrors`` option estimated the genotyping error rate based on observed information, this option is generally not necessary and can increase runtime. ``-estmaf`` estimates the minor allele frequency after each peeling cycle. This option can be useful if there are a large number of non-genotyped founders. 
 
 
+
```
Comment (gregorgorjanc) [inline docs/source/usage.rst:131]: @AprilYUZhang best to avoid adding these excessive empty lines;)

**Comment by gregorgorjanc** [comment]
@AprilYUZhang see some pedantic comments above;)

File: docs/source/usage.rst (hunk 1 above)
Comment (AprilYUZhang) [reply docs/source/usage.rst:126]: Ok, I will rewrite before Monday

File: docs/source/usage.rst (hunk 2 above)
Comment (AprilYUZhang) [reply docs/source/usage.rst:131]: Sorry, I'm not sure when I added it. I'll pay attention next time.

**Comment by gregorgorjanc** [comment]
Thanks @AprilYUZhang. Merging this in.

**Pull request merged**